templates/zerver/emails/organisation_info.txt

```diff
@@ -0,0 +1,13 @@
+{% extends "zerver/emails/compiled/email_base_default.html" %}
```

Comment:
Rename this file to `organization`.

Reply:
Also, I think this email really doesn't need a plain-text version, since it's just going to Zulip internal folks.